packages/alfa-rules/src/common/predicate/has-border.ts

```diff
@@ -0,0 +1,28 @@
+import { Color, Length } from "@siteimprove/alfa-css";
+import { Device } from "@siteimprove/alfa-device";
+import { Element } from "@siteimprove/alfa-dom";
+import { Predicate } from "@siteimprove/alfa-predicate";
+import { Context } from "@siteimprove/alfa-selector";
+import { Style } from "@siteimprove/alfa-style";
+
+const sides = ["top", "right", "bottom", "left"] as const;
```

Comment:
I think we should also check the logical `border-[block,inline]-[start,end]` in the same way since this won't cost much more and could be used.

Reply:
That should be handled by logical property mapping when implemented. Until then, I'd rather not have seemingly partial support for logical properties.